### PR TITLE
Add LRUG September 2024 meetup

### PR DESCRIPTION
**Pull request by fcheung**

### CODE REVIEW

File: _data/meetups.yml [hunk 1]
```diff
@@ -1437,6 +1437,13 @@
   end_time: 13:00:00 EAT
   url: https://www.meetup.com/african_ruby_community/events/300548245
 
+- name: London Ruby User Group: September 2024 Meeting
```
Comment (marcoroth) [inline _data/meetups.yml:1440]: Hey @fcheung, thanks for the PR!

If the value for a YAML column contains a `:` it sadly doesn't work. You either have to remove it, or wrap it in quotes. But if you want to match the style of the others I'd go with the dash:

```suggestion
- name: London Ruby User Group - September 2024 Meeting
```
or
```suggestion
- name: "London Ruby User Group: September 2024 Meeting"
```

**Comment by fcheung** [comment]
Oops - good call!

**Review by marcoroth — APPROVED** [review]
Thank you, @fcheung! 🙌🏼 

I was just looking at the ical [meeting calendar](https://lrug.org/meeting-calendar) to see if we can plug it in for #733.

**Comment by fcheung** [comment]
> Thank you, @fcheung! 🙌🏼
> 
> I was just looking at the ical [meeting calendar](https://lrug.org/meeting-calendar) to see if we can plug it in for #735.

That ical is a bit dodgy - it's just a calendar that @h-lame setup with a recurring meeting. I'm going to have a go at generating a proper calendar file

**Comment by marcoroth** [comment]
> I'm going to have a go at generating a proper calendar file from our meeting data

Oh, that would be great! if it helps, I opened these two Pull Requests to do the same thing for Seattle.rb and Toronto Ruby. Feel free to get some inspiration from those:

https://github.com/seattlerb/seattlerb.org/pull/203
https://github.com/toronto-ruby/toronto-ruby.com/pull/42

**Pull request merged**